iceoryx_examples/icedelivery/README.md

```diff
@@ -176,10 +173,10 @@ while (!iox::posix::hasTerminationRequested())
 }
 ```
 
-The `killswitch` will be used to stop the program execution.
+The program execution is stopped when the user presses `Ctrl-C`.
```

Comment:
We have to solve the issue if we are writing markdown documentation or mkdocs documentation for the website.

In my opinion this is github markdown documentation and therefore we should not enable those features.

But if we write mkdocs documentation for the website this is a nice feature, but then we should NOT directly link or provide the documentation via github and only direct to the website.

In the end the starting point of most developers will be github so I would see this is prio.

Reply:
I understand what you mean. However, for `v1.0` we decided to include the `.md` of all examples to avoid duplication work. Those `.md` files can be read via GitHub web GUI and as well on [iceoryx.io](https://iceoryx.io). We basically killed two birds with one stone.

I think most developers use iceoryx.io](https://iceoryx.io) to access the docs instead of viewing them via GitHub. I can dig out the click rates for both [iceoryx.io](https://iceoryx.io) and the access via GitHub if you're interested.

I would appreciate if we use the key feature, but it's not a blocker IMHO.